services/system/Accounts/ui/src/components/AccountSelection.tsx

```diff
@@ -0,0 +1,722 @@
+                                  <LoaderCircle
+                                    size={15}
+                                    className="animate animate-spin my-auto"
+                                  />{" "}
+                                </div>
+                              )}
+                            </FormLabel>
+                          )}
+                          {isDirty && !isCreatingAccount && (
+                            <FormLabel className="text-muted-foreground">
+                              {accountStatus === "Available" ? (
+                                <div className="flex gap-1">
+                                  <div className="text-sm">
+                                    Account does not exist
+                                  </div>
+                                  <UserX size={15} className=" my-auto" />{" "}
+                                </div>
+                              ) : accountStatus === "Invalid" ? (
+                                <div className="flex gap-1">
+                                  {" "}
+                                  <div className="text-sm">Invalid</div>
+                                  <UserX size={15} className=" my-auto" />{" "}
+                                </div>
+                              ) : accountStatus === "Loading" ? (
+                                <div className="flex gap-1">
+                                  {" "}
+                                  <div className="text-sm">Loading</div>
+                                  <LoaderCircle
+                                    size={15}
+                                    className="animate animate-spin my-auto"
+                                  />{" "}
+                                </div>
+                              ) : undefined}
+                            </FormLabel>
+                          )}
+                        </div>
+                        <FormControl>
+                          <Input {...field} />
+                        </FormControl>
+                        <FormMessage />
+                      </FormItem>
+                    )}
+                  />
+                  <Button type="submit" disabled={isSubmitting}>
+                    {isCreatingAccount
+                      ? `${isSubmitting ? "Accepting" : "Accept"} invite`
+                      : `Import account`}
+                  </Button>
+                </form>
+              </Form>
+            </DialogHeader>
+          </DialogContent>
+          <div className="max-w-lg mx-auto">
+            <div className="text-center text-muted-foreground py-2">
+              <span>
+                {isInitialLoading
+                  ? "Loading..."
+                  : isNoAccounts
+                  ? `No accounts available.`
+                  : isInvite
+                  ? `Select an account to accept invite to `
+                  : `Select an account to login to `}
```

Comment:
I don't think "No accounts available" is an alternative to 
"Select an account to accept invite to" and "Select an account to login to"

For example, in this case:
![image](https://github.com/user-attachments/assets/68e7234c-3dd9-4cb8-bf03-23874b01d4c2)

Probably should still say, "Select an account to (accept invite / login) to <appname>", no?